Updates to conclusion (#29)
thesis20/thesis_paper: sections/conclusion/conclusion-chapter.tex

\section{Conclusion}\label{sec:conclusion}
In this paper, we have shown two ways to include context and side-information in graph convolution neural networks.
While there are not many competing methods available for comparison in context specific recommendation scenarios, we have shown that compared to both traditional factorization machines and deep learning versions of those, the CSGCN models outperform them across most datasets.
Likewise, we have shown how the context can be aggregated and used in a general recommendation scenario with competitive results to state-of-the-art methods.
\\\\
There are, however, some limitations to the methods.

> We have observed that on the Frappe dataset, the CSGCN methods are being consistently outperformed by simple methods such traditional factorization machines and TopPop.

Reviewer: ```suggestion
We have observed that on the Frappe dataset, the CSGCN methods are being consistently outperformed by simple methods such as traditional factorization machines and TopPop.
```

\\
Additionally, we have seen that not all context is equal in terms of importance.
Due to this, it could be beneficial to include an attention mechanism that is able to learn the importance of each context or context combination for the convolution layer.
\\
Finally, both CSGCN methods are based on the simplified models of GCNs which are gaining popularity.

> For this paper, we have done limited experimentation on extending this with various components from traditional GCNs, such as re-adding linear transformation and non-linear activation, without success.

Reviewer 2: Viser vi egentlig det ngoel stedet med de her limited experiments? Ville det ikke være interessant?

> For this paper, we have done limited experimentations with extending this with various components from traditional GCNs, such as re-adding linear transformation and non-linear activation, without success.

Reviewer: ```suggestion
For this paper, we have done limited experimentatio on extending this with various components from traditional GCNs, such as re-adding linear transformation and non-linear activation, without success.
```

> In the recent months, several new papers on GCNs have been published, which present new extensions to GCNs.

Reviewer: ```suggestion
In the recent months, several new papers on GCNs have been published which present new extensions to GCNs.
```
